src/lib/util.py

```diff
@@ -3,10 +3,43 @@
 import os
 import subprocess
 
+def config_secret_sanitizer(str1):
+  ''' run all config secret sanitizers '''
+  artsy_sanitized = config_secret_sanitizer_artsy(str1)
+  eso_sanitized = config_secret_sanitizer_eso(artsy_sanitized)
+  return eso_sanitized
+
+def config_secret_sanitizer_artsy(str1):
+  ''' ensure secret_value conforms to Artsy requirements '''
+  # strip surrounding quotes if any
+  return unquote(str1)
```

Comment:
i doubt our our config secrets are currently quoted.

Reply:
if any existing secret has surrounding quotes, this sanitizer removes the quotes. on second thought, is it better to raise an exception instead?

Reply:
Not sure. We can scan our configmaps for double quotes pretty easily (like in the JWT-scanning script?).

Reply:
we are good. added [--dry-run](https://github.com/artsy/opstools/pull/65/commits/f836e711c14fd1ce7d42fcde9e7ccbfc13b826c9) which together with `--loglevel DEBUG` will tell us if a var's value is quoted:
```
(opstools) artsy:opstools jxu$ foreman run --env .env.shared,.env python src/migrate_config_secrets/migrate.py staging hokusai-sandbox  ~/code --list hokusai-sandbox_staging_secret_vars.txt --dry-run --loglevel DEBUG               
INFO:root:Migrating staging hokusai-sandbox sensitive configs from k8s configmap to Vault...
INFO:root:Getting list of sensitive vars...
INFO:root:Configuring vars in Vault...            
DEBUG:root:Updating bar value in Vault...  
...
DEBUG:root:string is quoted with ", removing quotes               
...
INFO:root:Skipping the rest because this is a dry run.
```